selenium-system-test/src/test/scala/hts/pages/AuthorityWizardPage.scala

```diff
@@ -17,26 +17,46 @@
 package hts.pages
 
 import hts.utils.Configuration
-import org.openqa.selenium.{By, WebDriver}
+import org.openqa.selenium.By
 
-object AuthorityWizardPage {
+object AuthorityWizardPage extends WebPage {
 
-  def goToPage()(implicit driver: WebDriver): Unit =
+  var tempNino: String = ""
```

Comment:
do we really need to store the nino here - the only place it's being set is `authenticateUser` where the nino is already known. Is there a reason why the code calling this object will have lost that nino in between `authenticateUser` and `getNino`?